jpx/src/main/java/io/jenetics/jpx/Length.java

```diff
@@ -209,24 +211,34 @@ public String toString() {
 	 * Create a new {@code Length} object with the given length.
 	 *
 	 * @param length the length
-	 * @param unit the length unit
+	 * @param unit   the length unit
 	 * @return a new {@code Length} object with the given length.
 	 * @throws NullPointerException if the given length {@code unit} is
-	 *         {@code null}
+	 *                              {@code null}
 	 */
 	public static Length of(final double length, final Unit unit) {
 		requireNonNull(unit);
 		return new Length(Unit.METER.convert(length, unit));
 	}
 
 	static Length parse(final String value) {
-		final String length = Strings.trim(value);
+		final Double length = parseDouble(value);
 
 		return length != null
-			? Length.of(Double.parseDouble(length), Unit.METER)
+			? Length.of(length, Unit.METER)
 			: null;
 	}
 
+	private static Double parseDouble(final String value) {
+		NumberFormat formatter = NumberFormat.getNumberInstance(ENGLISH);
+		try {
+			return formatter.parse(Strings.trim(value)).doubleValue();
+		} catch (ParseException e) {
+			e.printStackTrace();
```

Comment:
Should throw `NumberFormatException`, because this is thrown by the original parsing method, `Double::parseDouble`.